cmd/honeytrap/main.go

```diff
@@ -76,6 +76,11 @@ var globalFlags = []cli.Flag{
 		Value: "config.toml",
 		Usage: "Load configuration from `FILE`",
 	},
+	cli.StringFlag{
+		Name:  "data, d",
+		Value: ".honeytrap",
```

Comment:
Shouldn't this be `~/.honeytrap`?

Reply:
This doesn't play well with the `filepath.Abs(s)` function:
`mkdir /home/sille/go/src/github.com/honeytrap/honeytrap/~/.honeytrap: no such file or directory`

I could check for the `~/` value and use the currently existing `path.Join(usr.HomeDir, s)` so it works every time `~/` is used?